docs/debugging.rst

```diff
@@ -60,6 +60,21 @@ When an application is built, ``flatpak-builder`` automatically
 creates a ``.Debug`` extension. This can be disabled with the ``no-debuginfo``
 option.
 
+To install the ``.Debug`` extension created locally, export the build
+to a repo by passing ``--repo repo`` to ``flatpak-builder`` when
+building. Then add the repo folder as a local remote::
+
+  $ flatpak remote-add --user --no-gpg-verify test-debug file:///path/to/repo
+
+Then install the package from that remote::
+
+  $ flatpak install --user test-debug $FLATPAK_ID.Debug
+
+Alternatively, you can run ``flatpak install`` directly on the
+flatpak-builder cache folder::
+
+  $ flatpak install --user /absolute/path/to/.flatpak-builder/cache $FLATPAK_ID.Debug
```

Comment:
again with the package installed with `flatpak-builder --install` you don't need this either.